Scarb.lock

```diff
@@ -0,0 +1,6 @@
+# Code generated by scarb DO NOT EDIT.
+version = 1
+
+[[package]]
+name = "openzeppelin"
+version = "0.7.0"
```

Comment:
```suggestion
version = "0.8.0-beta.0"
```
we need to update this, although apparently it should be autogenerated. was this file introduced in scarb `2.3.0`?

Reply:
The file is autogenerated by scarb, since version 2.3.0, yes